Azure/Test/resource_virtual_network.tf

```diff
@@ -45,13 +45,19 @@ resource "azurerm_subnet" "aks" {
   resource_group_name  = azurerm_resource_group.vnet.name
   virtual_network_name = azurerm_virtual_network.vnet.name
   address_prefixes     = [var.sub_aks]
+
+  service_endpoints    = ["Microsoft.Sql"
+                          ]
 }
 
 resource "azurerm_subnet" "ehub" {
   name                 = "${var.project}-${var.environment}-${var.sub}-ehub"
   resource_group_name  = azurerm_resource_group.vnet.name
   virtual_network_name = azurerm_virtual_network.vnet.name
   address_prefixes     = [var.sub_ehub]
+
+  service_endpoints    = ["Microsoft.Sql"
```

Comment:
Don't think ehub needs it

Reply:
Removed eHub from config and added missing rule